### PR TITLE
style: add the German translation file.

### PR DESCRIPTION
**Pull request by DavidKlempfner**

### CODE REVIEW

**Comment by aguacongas** [comment]
#1197

File: src/Aguacongas.TheIdServer.Duende/Localization-de.json [hunk 1]
```diff
@@ -0,0 +1,2166 @@
...
+  {
+	"key": "Cannot have more than one URI per kind.",
+	"value": "Kann nicht mehr als eine URI pro Art haben."
+  },
+  {
+	"key": "Either a metadata or redirect URI must be set.",
+	"value": "Entweder eine Metadaten- oder Redirect-URI muss festgelegt sein."
+  },
+  {
+	"key": "Either a metadata or redirect URI must be set.",
```
Comment (aguacongas) [inline src/Aguacongas.TheIdServer.Duende/Localization-de.json:2163]: can you update this key with : "Either a metadata URI or a X509Certificate secret must be set."

**Review by aguacongas — COMMENTED** [review]
Can you update the Aguacongas.TheIdServer.Duende.csproj project file so this file will be deployed:

<ItemGroup>
  <Content Remove="Localization-fr.json" />
  <Content Remove="Localization-de.json" />
</ItemGroup>

<ItemGroup>
	<None Include="Localization-fr.json">
	  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
	</None>
	<None Include="Localization-de.json">
	  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
	</None>
</ItemGroup>

File: src/Aguacongas.TheIdServer.Duende/Localization-de.json (hunk 1 above)
Comment (DavidKlempfner) [reply src/Aguacongas.TheIdServer.Duende/Localization-de.json:2163]: Should I do the same in the -fr.json file?

Comment (aguacongas) [reply src/Aguacongas.TheIdServer.Duende/Localization-de.json]: I did, thx

**Comment by aguacongas** [comment]
#567

**Pull request merged**